cms/static/sass/views/_settings.scss

```diff
@@ -838,6 +838,184 @@
       }
     }
 
+    .add-new-data {
+      background-color: $green;
+      border-color: $green;
+      color: $white;
+      display: inline-block;
+      padding: 10px 20px;
```

Comment:
Instead of hard-coding numbers, I think that variables like $baseline should be used.